.php_cs

```diff
@@ -11,7 +11,7 @@ EOF;
 Symfony\CS\Fixer\Contrib\HeaderCommentFixer::setHeader($header);
 
 return Symfony\CS\Config\Config::create()
-    ->level('symfony')
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
Yeah I would not remove it too,  just saying why this worked (fallback to the default level I guess)